README.md

```diff
@@ -13,8 +13,8 @@ Code coverage: (develop branch)
 
 #### v2.1.2 - 3/16/18
   - Pull a new base image if possible each time a module is registered
-  - Fix extra_files logic
-  - Update NMS to use Auth2
+  - Fix the logic that allows additional html files to be passed from a method's 
```

Comment:
Sorry for being annoying about this, but we should say it's specifically for the validation step. There may be another step later that actually changes the backend DB after the image is built.

Reply:
Done